components/main-chef-wrapper/cmd/root.go

```diff
@@ -61,7 +61,7 @@ var (
 		// Stop framework from showing default errors. This prevents duplicate errors or
 		// unncessary info from showing on passthrough commands; and
 		// allows us control over error rendering for internal commands.
-		SilenceErrors: true,
```

Comment:
This comment is now out of date.  In addition, was there a specific type of error that we were suppressing that has become a problem? 

 The intent was to make sure the actual error is not obscured in front of a wall of usage text/instructions - it should be clearly shown as the final piece of output from a failed command whenever possible. 

If silencing errors caused other issues, we should look for an approach that allows them to be handled, but still doesn't show additional information after the error message - that's the most important thing for the user to see.

Reply:
The 2 issues I am facing if we are silencing the error is
1. When we are passing the invalid command at that time It fails in the  execute function  on   `rootCmd.Execute()`  call, without calling the chef-cli repo and exits the OS without any error.
2. Also in some cases where the commands don't accept the arguments so we are setting `DisableFlagParsing` to false. Now if I am passing any invalid flag to this command,  it will not parse this as an argument and checks for the passed invalid flag in the file and when it dosen't find it just exits the OS without any error when the execute func gets called..